presto-main/src/test/java/com/facebook/presto/operator/scalar/TestMathFunctions.java

```diff
@@ -313,6 +313,33 @@ public void testPow()
         assertFunction("pow(NULL, NULL)", DOUBLE, null);
         assertFunction("pow(5.0, NULL)", DOUBLE, null);
         assertFunction("pow(NULL, 5.0)", DOUBLE, null);
+
```

Comment:
I don't think we actually need all the tests for aliases. I think having a single assertion would be fine:
```java
// test alias
assertFunction("power(5, 3)", DOUBLE, 125.0);
assertFunction("pow(5, 3)", DOUBLE, 125.0);